package.json

```diff
@@ -97,7 +97,7 @@
     "typed-redux-saga": "1.5.0",
     "valid-url": "1.0.9",
     "webext-redux": "2.1.9",
```

Comment:
webext-redux v4 supports manifest v3 and service workers, but looking at PR we will remove this dep completely. What was an issue with using it or it was just simpler to get rid of it?